cst/plugins/eu.esdihumboldt.cst.functions.numeric/src/eu/esdihumboldt/cst/functions/numeric/sequentialid/SequentialID.java

```diff
@@ -78,16 +79,21 @@ protected Object evaluate(String transformationIdentifier, TransformationEngine
 			break;
 		}
 
-		long id;
+		long id = 1;
```

Comment:
I think it would make sense to remove the initialization here as that is never used.